Gemfile

```diff
@@ -82,6 +82,7 @@ group :test do
   gem 'chromedriver-helper'
   # Factories for creating database entities for testing
   gem 'factory_bot_rails'
+  gem 'rails-controller-testing'
```

Comment:
Since Rails 5 it is recommended to use the built in RSpec request tests, and not use the `rails-controller-testing` gem.

Reply:
I reverted the commit.